src/main/java/pl/patrycja/ox/gameflow/AutomaticTests.java

```diff
@@ -0,0 +1,82 @@
+package pl.patrycja.ox.gameflow;
+
+import pl.patrycja.ox.GameSettings;
+import pl.patrycja.ox.Sign;
+import pl.patrycja.ox.ui.UI;
+
+import java.util.List;
+
+class AutomaticTests extends Mode {
+
+    AutomaticTests(UI ui) {
+        super(ui);
+    }
+
+    @Override
+    public List<Player> createPlayers() {
+        Player firstPlayer = new Player("Henio", Sign.CROSS);
+        Player secondPlayer = new Player("Olaf", Sign.NAUGHT);
+        return List.of(firstPlayer, secondPlayer);
+    }
+
+    @Override
+    public void settings(String[] inputArrayParameters) {
+        gameSettings = setup(inputArrayParameters);
+        Sequence sequence = new Sequence(gameSettings, inputArrayParameters[2]);
+        sequence.generateSequence();
+    }
+
+    private GameSettings setup(String[] inputArrayParameters) {
+        checkCorrectInputData(inputArrayParameters);
+        //TODO: generate draw sequence!!
+        return updateValuesInSettings(inputArrayParameters);
+    }
+
+    private GameSettings updateValuesInSettings(String[] inputArrayParameters) {
+        GameSettings.GameSettingsBuilder gameSettings = GameSettings.builder();
```

Comment:
Could you collect all lines of builder into return statement and build it then?
Also this is not `gameSettings` but `gameSettingsBuilder` - not important when you will apply first line of this comment.

Reply:
Ok, I will move it to the return statement.